Mathlib/Analysis/Analytic/Linear.lean

```diff
@@ -30,20 +31,37 @@ namespace ContinuousLinearMap
 theorem fpowerSeries_radius (f : E →L[𝕜] F) (x : E) : (f.fpowerSeries x).radius = ∞ :=
   (f.fpowerSeries x).radius_eq_top_of_forall_image_add_eq_zero 2 fun _ => rfl
 
+protected theorem hasFiniteFPowerSeriesOnBall (f : E →L[𝕜] F) (x : E) :
+    HasFiniteFPowerSeriesOnBall f (f.fpowerSeries x) x 2 ∞ where
+  r_le := by simp
+  r_pos := ENNReal.coe_lt_top
+  hasSum := fun _ => (hasSum_nat_add_iff' 2).1 <| by
+    simp [Finset.sum_range_succ, ← sub_sub, hasSum_zero, fpowerSeries]
+  finite := by
+    intro m hm
+    cases' m with m
```

Comment:
I did not follow carefully but I think there was a recent effort to remove `cases'` from the library. Can you replace this (and the one below) with something else? (perhaps `obtain` or `rcases`?)

Reply:
Ah, yes. Changed to `match`.